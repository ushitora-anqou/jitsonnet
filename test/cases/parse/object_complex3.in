{
  local a = "b",
  ["foo"]: "bar",
  x: 1.0,
}
