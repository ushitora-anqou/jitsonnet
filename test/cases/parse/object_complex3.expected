{ Syntax.v =
  (Syntax.Object
     (Syntax.ObjectMemberList
        [(Syntax.MemberObjlocal
            (Syntax.Bind
               ("a",
                { Syntax.v = (Syntax.String "b");
                  loc =
                  (Some { Syntax.startpos =
                          { Syntax.fname =
                            "test/cases/parse/object_complex3.in"; line = 2;
                            column = 12 };
                          endpos =
                          { Syntax.fname =
                            "test/cases/parse/object_complex3.in"; line = 2;
                            column = 15 }
                          })
                  })));
          (Syntax.MemberField
             (Syntax.Field
                ((Syntax.FieldnameExpr
                    { Syntax.v = (Syntax.String "foo");
                      loc =
                      (Some { Syntax.startpos =
                              { Syntax.fname =
                                "test/cases/parse/object_complex3.in";
                                line = 3; column = 3 };
                              endpos =
                              { Syntax.fname =
                                "test/cases/parse/object_complex3.in";
                                line = 3; column = 8 }
                              })
                      }),
                 false, (Syntax.H 1),
                 { Syntax.v = (Syntax.String "bar");
                   loc =
                   (Some { Syntax.startpos =
                           { Syntax.fname =
                             "test/cases/parse/object_complex3.in"; line = 3;
                             column = 11 };
                           endpos =
                           { Syntax.fname =
                             "test/cases/parse/object_complex3.in"; line = 3;
                             column = 16 }
                           })
                   })));
          (Syntax.MemberField
             (Syntax.Field
                ((Syntax.FieldnameID
                    { Syntax.v = "x";
                      loc =
                      (Some { Syntax.startpos =
                              { Syntax.fname =
                                "test/cases/parse/object_complex3.in";
                                line = 4; column = 2 };
                              endpos =
                              { Syntax.fname =
                                "test/cases/parse/object_complex3.in";
                                line = 4; column = 3 }
                              })
                      }),
                 false, (Syntax.H 1),
                 { Syntax.v = (Syntax.Number 1.);
                   loc =
                   (Some { Syntax.startpos =
                           { Syntax.fname =
                             "test/cases/parse/object_complex3.in"; line = 4;
                             column = 5 };
                           endpos =
                           { Syntax.fname =
                             "test/cases/parse/object_complex3.in"; line = 4;
                             column = 8 }
                           })
                   })))
          ]));
  loc =
  (Some { Syntax.startpos =
          { Syntax.fname = "test/cases/parse/object_complex3.in"; line = 1;
            column = 0 };
          endpos =
          { Syntax.fname = "test/cases/parse/object_complex3.in"; line = 5;
            column = 1 }
          })
  }